### PR TITLE
Add Public Cemetery

### PR DESCRIPTION
**Pull request by pisskop**
Adds in a 6x5 public cemetery.

Its mostly stone gravestones in lines, and a few larger buildings that house empty coffins.
Throughout it the itemlist "grave_memories" spawns with low frequency
In the middle, a chance for an added item, "lucy's locket' in a mausoleum.

Adds 2 melee weapons, an iron rod and a sharpened ion rod.  neither is a fantastic weapon, due to their size and weight, but they are certianly serviceable.

Adds a recipe for said sharpened rod.  The dull rod can be obtained by smashing the iron fence, which requires a hulk or a car.  or maybe an explosion.

In the future, I can add bones into the coffins, but for now this should suffice.

### CODE REVIEW

**Comment by Mecares** [comment]
Do shovels spawn there?

**Comment by pisskop** [comment]
yea.  But you should know that most cemeteries do more craning and scooping and less digging with a shovel.

**Comment by Mecares** [comment]
I can accept a certain degree of unrealism as long it is plausible and increases fun

**Comment by Rivet-the-Zombie** [comment]
>an added item, "lucy's locket'

Got some plans for this involving some greater purpose, or is it just for fun?

**Comment by Mecares** [comment]
Also perfect place to run into the not yet existing Skelebrutes and Bonehulks.

File: data/json/items/armor.json [hunk 1]
```diff
@@ -6977,6 +6977,23 @@
     },
     {
         "type" : "ARMOR",
+        "id" : "locket_lucy",
```
Comment (mugling) [inline data/json/items/armor.json]: has no actual use especially considering we already have a number of items of jewelry

File: data/json/items/melee.json [hunk 2]
```diff
@@ -1952,5 +1952,40 @@
     "to_hit": 1,
     "category" : "weapons",
     "qualities": [["CUT", 1], ["BUTCHER", 8]]
-  }
+  },{
+        "type":"GENERIC",
```
Comment (mugling) [inline data/json/items/melee.json]: too similar to rebar/sharpened rebar

**Comment by mugling** [comment]
> In the future, I can add bones into the coffins

Would be best as part of this PR. Bones actually have a number of crafting usages.

> I can accept a certain degree of unrealism as long it is plausible and increases fun

Presence of shovel is entirely plausible

**Comment by pisskop** [comment]
> too similar to rebar/sharpened rebar

Id remove the sharpened part, though I cant imagine an iron fence that yields rebar or nothing.

**Comment by pisskop** [comment]
If youre willing to graverob theres plenty of bones now.

**Comment by pisskop** [comment]
It occurs to me that I should save the editting of the coffin for another pr after this clears.

Is there anything else that needs to be addressed?

**Comment by pisskop** [comment]
Going to close and reevaluate this, due to stagnation

**Pull request closed without merging**